### PR TITLE
API level 34 added on tests

### PR DESCRIPTION
**Pull request by juanky201271**
new Android API level 34 added.

### CODE REVIEW

File: scripts/integration_tests.sh [hunk 1]
```diff
@@ -8,7 +8,7 @@ set_api_target=false
 intel_host_os=true
 create_snapshot=false
 test_name_default="OfflineTestSuite"
-valid_api_levels=("23" "24" "25" "26" "27" "28" "29" "30" "31" "32" "33")
+valid_api_levels=("23" "24" "25" "26" "27" "28" "29" "30" "31" "32" "33", "34")
```
Comment (Oscar-Pepper) [inline scripts/integration_tests.sh]: remove comma

File: scripts/emulate_app.sh [hunk 2]
```diff
@@ -7,7 +7,7 @@ set_abi=false
 set_api_level=false
 set_api_target=false
 create_snapshot=false
-valid_api_levels=("23" "24" "25" "26" "27" "28" "29" "30" "31" "32" "33")
+valid_api_levels=("23" "24" "25" "26" "27" "28" "29" "30" "31" "32" "33", "34")
```
Comment (Oscar-Pepper) [inline scripts/emulate_app.sh]: remove comma

File: scripts/emulator_target.sh [hunk 3]
```diff
@@ -8,7 +8,7 @@ set_abi=false
 set_api_level=false
 set_api_target=false
 create_snapshot=false
-valid_api_levels=("23" "24" "25" "26" "27" "28" "29" "30" "31" "32" "33")
+valid_api_levels=("23" "24" "25" "26" "27" "28" "29" "30" "31" "32" , "34")
```
Comment (Oscar-Pepper) [inline scripts/emulator_target.sh]: remove comma and add api 33 also?

**Review by Oscar-Pepper — CHANGES_REQUESTED** [review]

File: scripts/integration_tests.sh (hunk 1 above)
Comment (juanky201271) [reply scripts/integration_tests.sh]: OMG..................... thanks. I'm losing my only neuron...

**Review by Oscar-Pepper — APPROVED** [review]

**Pull request merged**